lottie/src/main/java/com/airbnb/lottie/manager/ImageAssetManager.java

```diff
@@ -121,6 +121,7 @@ public void setDelegate(@Nullable ImageAssetDelegate assetDelegate) {
       Logger.warning("Unable to decode image.", e);
       return null;
     }
+    if (bitmap == null) return null;
```

Comment:
Can you add a warning here just like the IllegalArgumentException branch above?

Reply:
Yes, It's necessary.